ansible/roles/owncloud/defaults/main.yml

```diff
@@ -2152,7 +2154,7 @@ owncloud__apache__dependent_vhosts:
 # :program:`nginx` server configuration managed by the :ref:`debops.nginx` role.
 owncloud__nginx__dependent_servers:
 
-  ## https://docs.nextcloud.com/server/13/admin_manual/installation/nginx.html
```

Comment:
Isn't latest 18 and not 17?

Reply:
Nextcloud 18.0 is indeed the latest release, but I've had more time testing the role with Nextcloud 17.0, which is what I now use for my own systems. Nextcloud 17.0 is currently in the 'production' update channel. This means that it is considered suitable for enterprise use as no potentially breaking changes will be introduced anymore, which in turn makes it a favourable choice for system administrators who happen to choose Debian for its stability presumption. Nextcloud 17.0 is therefore a safe default for this role.

See also [DEP 3](https://docs.debops.org/en/master/dep/dep-0003.html).

The Nextcloud 19.0 release is scheduled for May 21st. We can do some work around that time to make Nextcloud 18.0 the default for this role, and start preparing things for the next major release.

Reply:
My intention was to update the documentation link to ``17` from ``latest``.

Reply:
Ah, check. I just pushed a commit to update the links to upstream documentation (version 17 for Nextcloud, 10.3 for ownCloud).